charts/opentelemetry-kube-stack/values.yaml

```diff
@@ -511,3 +511,157 @@ instrumentation:
   #   limits:
   #     memory: "128Mi"
   #     cpu: "500m"
+
+# Bridge cluster role configuration
+opAMPBridgeClusterRole:
+  # Whether the bridge cluster role is enabled or not
+  enabled: true
```

Comment:
Should this be enabled by default?

Reply:
I'm going to move this under the bridge at which point it being enabled makes sense as the bridge will be default disabled.